digitalpalireader/content/js/xml.js

```diff
@@ -964,26 +965,44 @@ function loadXMLindex(place,compare) {
 	saveout = '<div id="menu">'+saveheader.replace(clfirst,"$1<div class=\"csection\">$2").replace(clinner,"$1</div>$2").replace(cllast,"$1</div>")+'</div>'+saveout;
 
 	if(isDev)
-		return [tabT,saveout];
+    return [tabT,saveout];
 
-  $('#mafbc').html(`<nav class="navbar navbar-expand-lg navbar-light bg-light">${DPRSidebarHamburgerMenu()}`+`<span class="navbar-brand mb-0">Reading :</span>`+tabT+`</nav>`);
+  if (DPR_PAL.isWeb){
+    $('#mafbc').html(`<nav class="navbar navbar-expand-lg navbar-light bg-light">${DPRSidebarHamburgerMenu()}`+`<span class="navbar-brand mb-0">Reading :</span>`+tabT+`</nav>`);
+  }
   $('#mafbc').append('<div id="savetitle">'+tabT+'</div>');
-	$('#mafbc').append('<div id="savei">'+saveout+'</div>');
+  $('#mafbc').append('<div id="savei">'+saveout+'</div>');
   $('#mafbc').append('<div id="convi">'+convout+'</div>');
 
+
 	// title, tab, link
 
 	document.getElementsByTagName('title')[0].innerHTML = tabT;
 
-
-
-
   if (DPR_PAL.isWeb){
     theDatao=digitalpalireader.makeWebAppropriate(theDatao);
+    $('#mafbc').append('<div id="paliTextContent">	'+theDatao+'</div>');
+    DPR_PAL.setPaliTextContentHeight();
+  } else {
+    // permalink
+    var permalink = `${DPR_PAL.dprHomePage}?`;
```

Comment:
This else case is of course a PITA. We'll get rid of all xul stuff right after alpha.

Reply:
Ok

Reply:
Ok